src/core/int128.d

```diff
@@ -0,0 +1,695 @@
+/* 128 bit integer arithmetic.
+ *
+ * Not optimized for speed.
+ *
+ * Copyright: Copyright D Language Foundation 2022.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Walter Bright
+ * Source:    $(DRUNTIMESRC core/_int128.d)
+ */
+
+module core.int128;
+
+nothrow:
+@safe:
+@nogc:
+
+alias I = long;
+alias U = ulong;
+enum Ubits = U.sizeof * 8;
+
+struct Cent
```

Comment:
this is a `ucent`

Reply:
ucent is a keyword, cannot use that as a struct name.

Reply:
Also, the Cent struct will work with both signed and unsigned operations.